### ISSUE TITLE
version 4.7.1 avalible

### ISSUE THREAD

**Issue by masterwishx** [bitnami/bitnami-docker-matomo#102]
version 4.7.1 avalible in official docker, but we have 4.6.2 
is any specific reason for this ?

**Comment by carrodher** [comment]
Thanks for letting us know. Our automated test & release pipeline already detected the new version but it is failing during our internal tests, probably something changed in this new version and tests need to be adapted. The team will work on this manual change and the new version will be released as usual.

As we handle a large catalog, I can't provide an ETA since it is going to depend on the number of new versions requiring manual intervention.

**Comment by gongomgra** [comment]
We have just released the Matomo container for version `4.7.1`. You can pull it from the official registries using the command `docker pull bitnami/matomo:4.7.1`. 

```
$ docker pull bitnami/matomo:4.7.1
4.7.1: Pulling from bitnami/matomo
a2b89359fa38: Already exists
0a5830e618ce: Pull complete
f4d2dcccf42b: Pull complete
feab48d4e6b7: Pull complete
9e2c73d72768: Pull complete
3dbf43a3a379: Pull complete
bc9c1f3e633d: Pull complete
418194f39f0b: Pull complete
c903c2a15ec9: Pull complete
b887b684a3df: Pull complete
482513daf1e7: Pull complete
b0e2415c533c: Pull complete
3fb184134bd8: Pull complete
b17c50c04ce7: Pull complete
919b32cfa84c: Pull complete
709b92497c48: Pull complete
ff21e1faf3cc: Pull complete
4dd1022ec634: Pull complete
3ec168cc1175: Pull complete
a4b579bb7334: Pull complete
Digest: sha256:6ccea6303d020c31c5ce3140a8bcab22bd90506d5e09057b375978ea0a6f386c
Status: Downloaded newer image for bitnami/matomo:4.7.1
docker.io/bitnami/matomo:4.7.1
```

I'm also closing this ticket as solved. Thanks for using our products.
